doc/source/sparse.rst

```diff
@@ -15,13 +15,13 @@
 Sparse data structures
 **********************
 
-We have implemented "sparse" versions of Series, DataFrame, and Panel. These
-are not sparse in the typical "mostly 0". You can view these objects as being
-"compressed" where any data matching a specific value (NaN/missing by default,
-though any value can be chosen) is omitted. A special ``SparseIndex`` object
-tracks where data has been "sparsified". This will make much more sense in an
-example. All of the standard pandas data structures have a ``to_sparse``
-method:
+We have implemented "sparse" versions of Series and DataFrame (there used to be
+one for Panel but was removed in 0.19.0). These are not sparse in the typical
```

Comment:
put it in a ``warning`` or ``note``, just trying to draw attention. It should be the first item in that section.